test/common/dashboards_exist.go

```diff
@@ -151,7 +205,14 @@ func TestIntegreatlyDashboardsExist(t *testing.T, ctx *TestingContext) {
 	}
 }
 
-func getExpectedDashboard(installType string) []dashboardsTestRule {
+func getExpectedCustomerDashboard(installType string) []dashboardsTestRule {
+	if installType == string(integreatlyv1alpha1.InstallationTypeManagedApi) {
+		return customerRHOAMDashboards
+	}
+	return nil
+}
+
+func getExpectedMiddlewareDashboard(installType string) []dashboardsTestRule {
 	if installType == string(integreatlyv1alpha1.InstallationTypeManagedApi) {
 		return commonExpectedDashboards
 	} else {
```

Comment:
@laurafitzgerald would having a single function for this work, as we're passing in the same `installType` into both functions? If we add any additional install types in the future we might be able to just add an extra case statement. Maybe this isn't the best approach, but it's just an idea. Something like: 
```
func getExpectedDashboards(installType string) []dashboardsTestRule {
	switch installType {
	case string(integreatlyv1alpha1.InstallationTypeManagedApi):
		return append(customerRHOAMDashboards, commonExpectedDashboards...)
	case string(integreatlyv1alpha1.InstallationTypeManaged), string(integreatlyv1alpha1.InstallationTypeSelfManaged:
		return append(commonExpectedDashboards, rhmi2ExpectedDashboards...)
	}
	return append(commonExpectedDashboards, rhmi2ExpectedDashboards...)
}
```

Reply:
The returns are slightly different there. The first one is returning expected dashboards for the customer grafana and the second one for the middleware grafana.

Reply:
Yep I see. Thanks for explaining.